package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autorest/az",
-  "version": "1.3.4",
+  "version": "1.3.5",
```

Comment:
Don't think it's a good idea to bump the official version directlly without tried private release for at least several days.

Reply:
I have delivered preview branch to others, so it is not so urgent to release officially